src/libmetalldata/metall_graph.cpp

```diff
@@ -255,18 +255,16 @@ metall_graph::return_code metall_graph::ingest_parquet_edges(
     }
   }
 
-  auto metall_edges = m_pedges;
-
   size_t               local_num_edges = 0;
   size_t               local_num_nodes = m_pnode_to_idx->size();
   static metall_graph* sthis           = nullptr;
   sthis                                = this;
   parquetp.for_all(
     parquet_cols,
     [&](const std::vector<ygm::io::parquet_parser::parquet_type_variant>& row) {
-      auto rec = metall_edges->add_record();
+      auto rec = m_pedges->add_record();
```

Comment:
I didn't think you could reference instance members directly within a lambda. Are you sure this works?

Reply:
Seems to & passes tests.

Reply:
There may be nicer c++20 way to do it explicitly instead of capturing everything with &